hubblestack/utils/hubble_error.py

```diff
@@ -0,0 +1,24 @@
+"""
```

Comment:
We should have a single file for exceptions in Hubble.
Meed to migrate these classes in utils/exceptions.py later on.

Reply:
Yes, this will be done during merging salt-less